runtime/rasdump/javadump.cpp

```diff
@@ -1776,6 +1782,53 @@ JavaCoreDumpWriter::writeMemoryCountersSection(void)
 		"NULL\n"
 		"NULL           ------------------------------------------------------------------------\n"
 	);
+
+#if defined(J9ZTPF)
+	if (MAP_FAILED != mmap(NULL, 0, 0, MAP_SUPPORTED, 0, 0)) {
+		struct immap_privatemstats ztpfNativeStats;
+		int rc = mprivatestats(&ztpfNativeStats);
+
+		_OutputStream.writeCharacters("0SECTION       Native configuration information for z/TPF\n");
+		_OutputStream.writeCharacters("NULL           ==========================================\n");
+		if (0 != rc) {
+			_OutputStream.writeCharacters("1TPFCONFIG     Unavailable: ");
+			_OutputStream.writeInteger((UDATA)(IDATA)rc, "%zd\n");
+		} else {
+			_OutputStream.writeCharacters("1TPFHWMAXX     Highwater mark 64-bit heap 1MB frames (limited by MAXXMMES): ");
+			_OutputStream.writeInteger(ztpfNativeStats.totalMAXXMMESRegion1MBFrames, "%zu\n");
+			_OutputStream.writeCharacters("1TPFLIMMAXX    MAXXMMES keypoint A setting: ");
+			_OutputStream.writeInteger(ecbp2()->ce2proc->iproc_maxheap_64, "%zu\n");
+			_OutputStream.writeCharacters("1TPFHWMMAP     Highwater mark 64-bit MMAP heap 1MB frames (limited by MAXMMAP): ");
+			_OutputStream.writeInteger(ztpfNativeStats.totalMMAPRegion1MBFrames, "%zu\n");
+			_OutputStream.writeCharacters("1TPFLIMMMAP    MAXMMAP process setting (either keypoint A MAXMMAP or com.ibm.tpf.maxmmap property): ");
+			_OutputStream.writeInteger(ztpfNativeStats.currentProcessMAXMMAPLimit, "%zu\n");
+			_OutputStream.writeCharacters("1TPFTOTHW64    Total highwater mark 64-bit heap 1MB frames (limited by MAXXMMES+MAXMMAP): ");
+			_OutputStream.writeInteger(ztpfNativeStats.total64bitHeap1MBFrames, "%zu\n");
+			_OutputStream.writeCharacters("1TPFHWEMPS     Total highwater mark 31-bit heap 1MB frames (limited by EMPS): ");
+			_OutputStream.writeInteger(ztpfNativeStats.total31bitHeap1MBFrames, "%zu\n");
+			_OutputStream.writeCharacters("1TPFLIMEMPS    EMPS keypoint A setting: ");
+			_OutputStream.writeInteger(ecbp2()->ce2proc->iproc_maxheap_31, "%zu\n");
+			_OutputStream.writeCharacters("1TPFHWGC	      Total highwater mark GC heap 1MB frames (limited by -Xmx): ");
```

Comment:
Please replace the tab within the string literal with a suitable number of spaces.

Reply:
If we were referring to line 1811, I assume there was a tab with one space, I deleted all 'spaces' and added them back in.  Not sure how to view tabs in a source file using vim or git.

I think I fixed it.

Reply:
In vim, say ":set list" which will display tabs as `^I`; say ":set nolist" to normal presentation of tabs, etc.